cumulus/polkadot-parachain/Cargo.toml

```diff
@@ -51,83 +44,33 @@ testnet-parachains-constants = { features = [
-sp-keystore = { workspace = true, default-features = true }
 sc-chain-spec = { workspace = true, default-features = true }
-sc-rpc = { workspace = true, default-features = true }
-sp-version = { workspace = true, default-features = true }
-sc-tracing = { workspace = true, default-features = true }
-sp-offchain = { workspace = true, default-features = true }
-frame-system-rpc-runtime-api = { workspace = true, default-features = true }
-pallet-transaction-payment = { workspace = true, default-features = true }
-pallet-transaction-payment-rpc-runtime-api = { workspace = true, default-features = true }
-sp-std = { workspace = true, default-features = true }
-sp-inherents = { workspace = true, default-features = true }
-sp-api = { workspace = true, default-features = true }
-sp-consensus-aura = { workspace = true, default-features = true }
-sc-sysinfo = { workspace = true, default-features = true }
-prometheus-endpoint = { workspace = true, default-features = true }
-sc-transaction-pool-api = { workspace = true, default-features = true }
-substrate-frame-rpc-system = { workspace = true, default-features = true }
-pallet-transaction-payment-rpc = { workspace = true, default-features = true }
-substrate-state-trie-migration-rpc = { workspace = true, default-features = true }
 
 # Polkadot
 # Use rococo-native as this is currently the default "local" relay chain
-polkadot-cli = { features = ["rococo-native", "westend-native"], workspace = true, default-features = true }
-polkadot-primitives = { workspace = true, default-features = true }
 polkadot-service = { workspace = true, default-features = true }
 xcm = { workspace = true, default-features = true }
 
 # Cumulus
-cumulus-client-cli = { workspace = true, default-features = true }
-cumulus-client-collator = { workspace = true, default-features = true }
-cumulus-client-consensus-aura = { workspace = true, default-features = true }
-cumulus-client-consensus-relay-chain = { workspace = true, default-features = true }
-cumulus-client-consensus-common = { workspace = true, default-features = true }
-cumulus-client-consensus-proposer = { workspace = true, default-features = true }
-cumulus-client-parachain-inherent = { workspace = true, default-features = true }
-cumulus-client-service = { workspace = true, default-features = true }
-cumulus-primitives-aura = { workspace = true, default-features = true }
 cumulus-primitives-core = { workspace = true, default-features = true }
-cumulus-relay-chain-interface = { workspace = true, default-features = true }
 
 [build-dependencies]
 substrate-build-script-utils = { workspace = true, default-features = true }
 
-[dev-dependencies]
-assert_cmd = { workspace = true }
-nix = { features = ["signal"], workspace = true }
-tempfile = { workspace = true }
-tokio = { version = "1.32.0", features = ["macros", "parking_lot", "time"] }
-wait-timeout = { workspace = true }
-
 [features]
 default = []
 runtime-benchmarks = [
+	"cumulus-primitives-core/runtime-benchmarks",
```

Comment:
we probably can remove all those features. I don't really know why they exists at the first place

Reply:
If it's related to `runtime-benchmarks` I answered on a different comment below